src/App/Pages/Settings/OptionsPageViewModel.cs

```diff
@@ -184,19 +209,30 @@ public bool ShowAndroidAutofillSettings
         public async Task InitAsync()
         {
             AutofillSavePrompt = !(await _stateService.GetAutofillDisableSavePromptAsync()).GetValueOrDefault();
+
             var blockedUrisList = await _stateService.GetAutofillBlacklistedUrisAsync();
             AutofillBlockedUris = blockedUrisList != null ? string.Join(", ", blockedUrisList) : null;
+
             AutoTotpCopy = !(await _stateService.GetDisableAutoTotpCopyAsync() ?? false);
+
             Favicon = !(await _stateService.GetDisableFaviconAsync()).GetValueOrDefault();
+
             var theme = await _stateService.GetThemeAsync();
             ThemeSelectedIndex = ThemeOptions.FindIndex(k => k.Key == theme);
+
             var autoDarkTheme = await _stateService.GetAutoDarkThemeAsync() ?? "dark";
             AutoDarkThemeSelectedIndex = AutoDarkThemeOptions.FindIndex(k => k.Key == autoDarkTheme);
+
             var defaultUriMatch = await _stateService.GetDefaultUriMatchAsync();
             UriMatchSelectedIndex = defaultUriMatch == null ? 0 :
                 UriMatchOptions.FindIndex(k => (int?)k.Key == defaultUriMatch);
+
             var clearClipboard = await _stateService.GetClearClipboardAsync();
             ClearClipboardSelectedIndex = ClearClipboardOptions.FindIndex(k => k.Key == clearClipboard);
+
+            var appLocale = _synchronousStorageService.Get<string>(Core.Constants.AppLocaleKey);
```

Comment:
Using the storage service directly from the app without going through `_stateService` goes against the pattern established with multi-account support (even for globals).  We could add a `Synchronous` bool to `StorageOptions` and create a `GetDefaultSynchronousStorageOptionsAsync` method to `StateService` to be used from new AppLocale getter/setter methods, lining up nicely with the other app settings.  (I'm assuming this value is global since it sounds like Locale must be set on app initialization, so check out `GetThemeAsync` & `SetThemeAsync` in `StateService` for a good global reference)

Reply:
I'll make the change, although I think that at some point we should refactor it to have the globals in some other place and leave the `StateService` for account related stuff only. I believe the `StateService` is getting gigantic and having a lot of responsabilities.